eclass/qt5-build.eclass

```diff
@@ -30,6 +30,18 @@ if [[ ${PV} == *9999* ]]; then
 fi
 readonly QT5_BUILD_TYPE
 
+# @ECLASS-VARIABLE: QT5_KDEPATCHSET_NO
+# @DEFAULT_UNSET
```

Comment:
I wouldn't tie it to the revision number because revisions could happen for any reason. We didn't choose the snapshot style because it would complicate revdep version defs. And now it is back to (a lot of) patches on top of an upstream release anyway,